spec/integration/runner/variant-reassignment-example-15.spec.js

```diff
@@ -0,0 +1,98 @@
+ +---------+----------------------------+----------------------------+                    +----------------------------+------------------------------------------------------------+
+ | Staged  | Product:                   | Product:                   |                    | Product:                   | Product:id: 2                                              |
+ |         | slug: { en: "product" }    | id: 1                      |                    | id: 1                      | slug: { en: "product-${timestamp}", _ctsd: "${timestamp} } |
+ |         | product-type: "pt2"        | slug: { en: "product" }    |                    | slug: { en: "product" }    | product-type: "pt2"                                        |
+ |         | masterVariant: { sku: v1 } | product-type: "pt2"        |                    | product-type: "pt2"        | masterVariant: { sku: v5 }                                 |
+ |         |                            | masterVariant: { sku: v1 } |                    | masterVariant: { sku: v1 } |                                                            |
+ +---------+----------------------------+----------------------------+                    +----------------------------+------------------------------------------------------------+
+ | Current |                            | Product:                   |                    | Product:                   |                                                            |
+ |         |                            | id: 1                      |                    | id: 1                      |                                                            |
+ |         |                            | slug: { en: "product" }    |                    | slug: { en: "product" }    |                                                            |
+ |         |                            | product-type: "pt2"        |                    | product-type: "pt2"        |                                                            |
+ |         |                            | masterVariant: { sku: v5 } |                    | masterVariant: { sku: v1 } |                                                            |
+ +---------+----------------------------+----------------------------+--------------------+----------------------------+------------------------------------------------------------+
+ */
+/* eslint-enable max-len */
+describe('Variant reassignment', () => {
+  const logger = utils.createLogger(__filename)
+  let ctpClient
+  let product1
+
+  before(async () => {
+    ctpClient = await utils.createClient()
+
+    let product1Response = await utils.createCtpProducts([['5']], ctpClient)
+    product1 = product1Response[0]
+    product1Response = await ctpClient.products
+      .byId(product1.id)
+      .update({
+        version: product1.version,
+        actions: [{ action: 'publish' }]
+      })
+    product1Response = await ctpClient.products
```

Comment:
You can merge these three actions below into one request.

Reply:
https://github.com/commercetools/commercetools-node-variant-reassignment/pull/19/commits/e27316f3b6af91331943c71ba6ca28c1b582ea56